### PR TITLE
Update editor_controller.rb

### LINKED ISSUE #11502: replace "and" with "&&" editor-controller
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from underrepresented groups in free and open-source software!

We know that the process of creating a pull request is one of the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [app/controllers/editor_controller.rb](https://github.com/publiclab/plots2/blob/main/app%2Fcontrollers%2Feditor_controller.rb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -35,12 +35,12 @@ def rich
       @main_image = Image.find_by(id: params[:main_image]).path
     end
 
-    if params[:tags]&.include? "lat:" and params[:tags]&.include? "lon:"
+    if params[:tags]&.include?("lat:") && params[:tags]&.include?("lon:")
       tags = params[:tags].split(',')
 
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are not familiar with GitHub or the pull request model, [here is a guide you can follow](https://guides.github.com/activities/hello-world/) on how GitHub works.
2. If you are familiar with the terminal or would like to learn to use it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

3. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in the comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

### PR DESCRIPTION
**Pull request by AlphaRomeoMike**
Removed 'and', replaced the logic with double ampersand

Fixes #11502 🎉🍾

### CODE REVIEW

**Comment by IvyJeptoo** [comment]
@AlphaRomeoMike can I help you out with the tests?

**Comment by AlphaRomeoMike** [comment]
> @AlphaRomeoMike can I help you out with the tests?

Yes please

**Comment by IvyJeptoo** [comment]
> > @AlphaRomeoMike can I help you out with the tests?
> 
> Yes please

maybe you could send me a chat on the communication platform my username is briesta so that we can hop into a meet

**Comment by AlphaRomeoMike** [comment]
> > > @AlphaRomeoMike can I help you out with the tests?
> > 
> > 
> > Yes please
> 
> maybe you could send me a chat on the communication platform my username is briesta so that we can hop into a meet

I dont think I will be able to because of my busy schedule

**Comment by AlphaRomeoMike** [comment]
> > > > > @AlphaRomeoMike can I help you out with the tests?
> > > > 
> > > > 
> > > > Yes please
> > > 
> > > 
> > > maybe you could send me a chat on the communication platform my username is briesta so that we can hop into a meet
> > 
> > 
> > I dont think I will be able to because of my busy schedule
> 
> How do you suggest I help you?

Balaclava_Nerd#6566 on Discord

**Comment by IvyJeptoo** [comment]
> > > > > > @AlphaRomeoMike can I help you out with the tests?
> > > > > 
> > > > > 
> > > > > Yes please
> > > > 
> > > > 
> > > > maybe you could send me a chat on the communication platform my username is briesta so that we can hop into a meet
> > > 
> > > 
> > > I dont think I will be able to because of my busy schedule
> > 
> > 
> > How do you suggest I help you?
> 
> Balaclava_Nerd#6566 on Discord

Great, sent!(Ivy J)

File: app/controllers/editor_controller.rb [hunk 1]
```diff
@@ -35,12 +35,12 @@ def rich
       @main_image = Image.find_by(id: params[:main_image]).path
     end
 
-    if params[:tags]&.include? "lat:" and params[:tags]&.include? "lon:"
+    if params[:tags]&.include? "lat:" && params[:tags]&.include? "lon:"
```
Comment (TildaDares) [inline app/controllers/editor_controller.rb:38]:
```suggestion
    if params[:tags]&.include?('lat:') && params[:tags]&.include?('lon:')
```

File: app/controllers/editor_controller.rb [hunk 2]
```diff
@@ -35,12 +35,12 @@
...
-    if params[:tags]&.include? "lat:" and params[:tags]&.include? "lon:"
+    if params[:tags]&.include?('lat:') && params[:tags]&.include?('lon:')
       tags = params[:tags].split(',')
       tags.each do |x|
-        x.include? "lat:" and (@lat = x.split(':')[1])
-        x.include? "lon:" and (@lon = x.split(':')[1])
-        x.include? "zoom:" and (@zoom = x.split(':')[1])
+        x.include? "lat:" && (@lat = x.split(':')[1])
+        x.include? "lon:" && (@lon = x.split(':')[1])
+        x.include? "zoom:" && (@zoom = x.split(':')[1])
```
[the remark this answers is not in the document]
Comment (AlphaRomeoMike) [reply app/controllers/editor_controller.rb:43]: @TildaDares can you please review it once, LGTM!

**Review by TildaDares — APPROVED** [review]
Amazing work @AlphaRomeoMike. Thank you!

File: app/controllers/editor_controller.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (TildaDares) [reply app/controllers/editor_controller.rb:43]: Can you add the code suggestions codecov made? Thank you!

[the remark this answers is not in the document]
Comment (TildaDares) [reply app/controllers/editor_controller.rb:43]: Hi @AlphaRomeoMike, do you need help with this? Please reach out if you do!

**Comment by cesswairimu** [comment]
Looking great, thanks all, merging now.

@IvyJeptoo @AlphaRomeoMike would any of you like to open an issue and fix the code-climate warnings raised here?  you can find the suggestions here https://codeclimate.com/github/publiclab/plots2/pull/11507, Its similar changes @TildaDares  mentioned [above](https://github.com/publiclab/plots2/pull/11507#pullrequestreview-1150756467).

**Pull request merged**